resources/building-mobile-extensions/logging-from-extension.md

```diff
@@ -0,0 +1,44 @@
+# Logging from an extension
+
+We recommend using the Mobile SDK logging API in order to print any message from the Extension code. The Mobile SDK provides to the application developers the `setLogLevel`  API which is used for setting the Mobile SDK logging mode: VERBOSE, DEBUG, WARNING and ERROR. If current logging mode is less verbose, then the message will not be printed out by the Mobile SDK. When debugging and testing your extension, you or the application developer can make the logging more verbose in order to see all the messages flowing through the Mobile SDK.
```

Comment:
- Please reword/restructure this section so that it reads as follows:

"We recommend that you use the Mobile SDK logging API to print a message from the Extension code. The Mobile SDK provides the `setLogLevel` API, which is used to set one of the following Mobile SDK logging modes: 

* VERBOSE
* DEBUG
* WARNING
* ERROR

If the current logging mode is less verbose, the message is not printed by the Mobile SDK. When debugging and testing your extension, to see all the messages flowing through the Mobile SDK, you or the application developer can make the logging more verbose."